pkg/controllers/grafana-operator/controller.go

```diff
@@ -114,7 +117,7 @@ func RegisterWithManager(mgr ctrl.Manager) error {
 	c, err := controller.New("grafana-operator", mgr, controller.Options{
 		MaxConcurrentReconciles: 1,
 		Reconciler:              r,
-		Log:                     logger,
+		Log:                     ctrl.Log,
```

Comment:
The controller name will added as a prefix by the `controller.New` function, so we don't need to create a new logger with the name of the controller.